docs/sources/flow/reference/components/prometheus.operator.podmonitors.md

```diff
@@ -191,3 +197,26 @@ prometheus.operator.podmonitors "pods" {
     }
 }
 ```
+
+This example will apply additional relabel rules to discovered targets for hashmod sharding. Here we are have 5 shards, and an environment variable is set up to give our current agent's shard.
```

Comment:
```suggestion
This example will apply additional relabel rules to discovered targets for hashmod sharding. Here we have five shards and an environment variable providing our current agent's shard.
```

I'm not sure about the wording here. Is the env var (is that this part? `env("HASHMOD_SHARD")`) storing the name of the agent's shard?  or something else? Depending on what's happening here the wording can change slightly.

Reply:
Yeah, the idea is that you are running 5 agents, and each has a different value for HASHMOD_SHARD given.